src/Passes/docs/src/UserGuide/QuickStart.md

```diff
@@ -0,0 +1,102 @@
+# Quick start
+
+Before we start, you will need to build the QAT tool. To this end, create a new folder `Debug` and type `make qat`:
+
+```sh
+mkdir Debug
+cd Debug
+make qat
```

Comment:
Can you add a comment (for `make qat`) briefly explaining which files are used for building? 

Looking at the commands I have an impression that we are creating an empty dir (`Debug`), enter that dir, and do `make qat` which does not find any files (or any implicit rule for `qat`) and _fails_.

Reply:
You are right. There is missing a `cmake ..`. I've updated the document accordingly.